identity/src/main/java/com/android/identity/crypto/EcCurve.kt

```diff
@@ -96,4 +97,7 @@ enum class EcCurve(val coseCurveIdentifier: Int) {
             ED448 -> Algorithm.EDDSA
             X448 -> Algorithm.UNSET
         }
+
+    // helper to easily return the size of key in octets
```

Comment:
I think I would prefer not to include this (we also don't seem to use it below). Also, if we're adding public API to a library it needs KDoc.